docs/examples/viz_network_animated.py

```diff
@@ -122,8 +122,8 @@ def new_layout_timer(showm, edges_list, vertices_count,
     b = 1.0
     deltaT = 1.0
 
-    sphere_geometry = np.array(numpy_support.vtk_to_numpy(
-        sphere_actor.GetMapper().GetInput().GetPoints().GetData()))
+    sphere_geometry = np.array(get_polydata_vertices(
+        sphere_actor.GetMapper().GetInput()))
```

Comment:
`vertices_from_actor(sphere_actor).copy()` instead. you will have to import the function from `utils`

Reply:
done @skoudoro, PTAL
thanks